templates/granules.html

```diff
@@ -1,4 +1,4 @@
-<html>
+<html lang="en-us">
```

Comment:
I'll wire this up next, via fetch...or maybe I'll just have the endpoint return a redirect with data.